src/run/rolluptest.rs

```diff
@@ -0,0 +1,179 @@
+    let mut buf = vec![4u8];
+    buf.extend(height_l2.to_bytes_be());
+    buf.extend(claimer.to_bytes_be());
+    rt_env.insert_l1_message(8u8, Uint256::zero(), &buf);
+}
+
+pub fn _insert_rollup_debug(rt_env: &mut RuntimeEnvironment) {
+    rt_env.insert_l1_message(8u8, Uint256::zero(), &[255u8]);
+}
+
+pub fn _test_rollup_tracker() {
+    let rt_env = RuntimeEnvironment::_new_with_owner(Uint256::from_usize(1111), None);
+    let mut machine = load_from_file(Path::new("arb_os/arbos.mexe"), rt_env);
+    machine.start_at_zero();
+
+    let wallet = machine.runtime_env.new_wallet();
+    let owner = Uint256::from_bytes(wallet.address().as_bytes());
+    let arbowner = _ArbOwner::_new(&wallet, false);
+    arbowner._give_ownership(&mut machine, owner.clone(), Some(Uint256::zero())).unwrap();
+
+    let my_addr = Uint256::from_u64(11025);
+    let claimer = Uint256::from_u64(4242);
+
+    machine.runtime_env.insert_eth_deposit_message(
+        owner.clone(),
+        owner.clone(),
+        Uint256::_from_eth(1),
+    );
+    machine.runtime_env.insert_eth_deposit_message(
+        claimer.clone(),
+        claimer.clone(),
+        Uint256::_from_eth(1),
+    );
+
+    arbowner._add_to_reserve_funds(&mut machine, Uint256::_from_eth(1)).unwrap();
+
+    _insert_create_node(
+        &mut machine.runtime_env,
+        &Uint256::one(),
+        &Uint256::zero(),
+        None,
+        &Uint256::from_u64(10),
+        my_addr.clone(),
+    );
+    _insert_create_node(
+        &mut machine.runtime_env,
+        &Uint256::from_u64(2),
+        &Uint256::one(),
+        None,
+        &Uint256::from_u64(10),
+        my_addr.clone(),
+    );
+
+    _insert_claim_node(&mut machine.runtime_env, &Uint256::from_u64(2), &claimer);
+
+    _insert_create_node(
+        &mut machine.runtime_env,
+        &Uint256::from_u64(3),
+        &Uint256::one(),
+        None,
+        &Uint256::from_u64(10),
+        my_addr.clone(),
+    );
+
+    _insert_create_node(
+        &mut machine.runtime_env,
+        &Uint256::from_u64(4),
+        &Uint256::from_u64(2),
+        None,
+        &Uint256::from_u64(10),
+        my_addr.clone(),
+    );
+
+    _insert_new_stake(
+        &mut machine.runtime_env,
+        &Uint256::from_u64(4),
+        &claimer,
+        None,
+    );
+
+    _insert_rollup_debug(&mut machine.runtime_env);
+    println!("_");
+    let _ = machine.run(None);
+
+    _insert_confirm_node(&mut machine.runtime_env, &Uint256::zero());
+    _insert_confirm_node(&mut machine.runtime_env, &Uint256::one());
+    _insert_confirm_node(&mut machine.runtime_env, &Uint256::from_u64(2));
+    _insert_reject_node(&mut machine.runtime_env, &Uint256::from_u64(3));
+    _insert_confirm_node(&mut machine.runtime_env, &Uint256::from_u64(4));
+
+    println!("B");
+    machine
+        .runtime_env
+        ._advance_time(Uint256::from_u64(100), None, true);
+
+    println!("C");
```

Comment:
Looks like these prints are left over from testing and should likely be deleted

Reply:
Fixed.